src/main/java/com/microsoft/sqlserver/jdbc/SQLServerBulkCopy.java

```diff
@@ -1240,6 +1240,11 @@ private String getDestTypeFromSrcType(int srcColIndx,
             int destColIndx,
             TDSWriter tdsWriter) throws SQLServerException {
         boolean isStreaming;
+        /**
+         * The maximum temporal precision we can send when using varchar(precision) in bulkcommand, to send a smalldatetime/datetime 
+         * value.
+         */
+        int sourceBulkRecordTemporalMaxPrecision = 50;
```

Comment:
is value of sourceBulkRecordTemporalMaxPrecision constant? if so then can we use final modifier for mentioned variable? better class level `static final`